deltachat-ios/Chat/Views/Cells/VideoInviteCell.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+import UIKit
+import DcCore
+
+public class VideoInviteCell: UITableViewCell {
+
+    private lazy var messageBackgroundContainer: BackgroundContainer = {
+        let container = BackgroundContainer()
+        container.image = UIImage(color: DcColors.systemMessageBackgroundColor)
+        container.contentMode = .scaleToFill
+        container.clipsToBounds = true
+        container.translatesAutoresizingMaskIntoConstraints = false
+        return container
+    }()
+
+    lazy var avatarView: InitialsBadge = {
+        let view = InitialsBadge(size: 28)
+        view.setColor(DcColors.systemMessageBackgroundColor)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        return view
+    }()
+
+    lazy var videoIcon: InitialsBadge = {
+        let view = InitialsBadge(size: 28)
+        view.setColor(DcColors.systemMessageBackgroundColor)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        view.setImage(#imageLiteral(resourceName: "ic_videochat").withRenderingMode(.alwaysTemplate))
+        view.tintColor = DcColors.defaultInverseColor
+        return view
+    }()
+
+    lazy var messageLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.textAlignment = .center
+        label.font = UIFont.preferredFont(for: .body, weight: .regular)
+        return label
+    }()
+
+    lazy var openLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.textAlignment = .center
+        label.font = UIFont.preferredFont(for: .body, weight: .bold)
+        label.text = String.localized("open")
```

Comment:
the label should be the string `videochat_tap_to_open` for outgoing invites and `videochat_tap_to_join` for incoming invites. i suggest to set that together where `messageLabel.text` is set.